### DISCUSSION TITLE
Disable automatic backups possible?

### DISCUSSION THREAD

**Discussion by pcace** [nextcloud/all-in-one#4831] [Questions] [answered]
Hi there, 

is it possible to disable the automatic backup creation? 
and second: 
if the  automatic backup failed - is there any way to start the containers automaticvally after the failed backup? 

Thanks alot!
cheers

**Comment by szaimen** [comment, marked as answer]
> is it possible to disable the automatic backup creation?

Yes it is

**Reply by pcace** [reply]
how?

**Reply by szaimen** [reply]
Use this button: (encircled in red)
![image](https://github.com/nextcloud/all-in-one/assets/42591237/bb4a20ce-2840-4263-aff5-918fa0b7855f)

**Reply by pcace** [reply]
Interesting. this is how it looks for me:

![Bildschirmfoto vom 2024-06-17 11-24-40](https://github.com/nextcloud/all-in-one/assets/1011875/a04bf321-deac-46c2-9209-9dda8570eb7c)

![Bildschirmfoto vom 2024-06-17 11-24-45](https://github.com/nextcloud/all-in-one/assets/1011875/84a283eb-859b-4ab4-8c65-55dd51fcfdb9)

**Reply by szaimen** [reply]
So it looks like you have not enabled automatic updates.

**Reply by pcace** [reply]
ah ok, sorry then! I thought automatic backups are enabled per default!
Thank you!
